safehttp/incoming_request.go

```diff
@@ -47,11 +47,15 @@ func NewIncomingRequest(req *http.Request) *IncomingRequest {
 	if req == nil {
 		return nil
 	}
+	req = req.WithContext(context.WithValue(req.Context(),
+		flightValuesCtxKey{}, &flightValues{m: map[interface{}]interface{}{}}))
```

Comment:
You store a pointer to flightValues, but the methods Put and Get are defined on the non-pointer type. Is that on purpose?

Reply:
Changed to a value. Thanks!

Done in 0ce62ea.